package.json

```diff
@@ -144,13 +128,13 @@
 		"fixtures:regenerate": "npm run fixtures:clean && npm run fixtures:generate",
 		"package-plugin": "./bin/build-plugin-zip.sh",
 		"pot-to-php": "./bin/pot-to-php.js",
-		"test-unit": "wp-scripts test-unit-js",
+		"test-unit": "wp-scripts test-unit-js --config test/unit/jest.config.json",
```

Comment:
I'm not certain what's the difference between `wp-scripts test-unit-js` and just `jest`. Both seems to work which makes me wonder why we need this script at all cc @gziolo